plaso/engine/path_helper.py

```diff
@@ -133,3 +147,26 @@ def GetRelativePathForPathSpec(cls, path_spec, mount_path=None):
       location = location[len(mount_path):]
 
     return location
+
+  @classmethod
+  def ExpandUserHomeDirPath(cls, path, user_accounts):
```

Comment:
Windows calls this profile directory not home directory, also don's use abbreviations such as `Dir`. This is more clear for an international audience.

Reply:
This could be for any operating system.  Removed abbreviation.